.github/workflows/bump.yaml

```
@@ -0,0 +1,47 @@
+---
+name: Bump
+
+on:
+  push:
+    branches:
+      - '**'
+  workflow_dispatch: # Manual workflow trigger
+
+jobs:
+  bump-so-version:
+    name:
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+    steps:
+      - name: Setup Golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.20.x
```

I'd go with `stable` to have it always on a floating version to avoid bumps in the future.  
```suggestion
          go-version: stable
```

---

stable seems a bit vague? I do like explicit better

---

I like explicit versions too